iina/InitialWindowController.swift

```diff
@@ -28,8 +29,16 @@ class InitialWindowController: NSWindowController {
   @IBOutlet weak var visualEffectView: NSVisualEffectView!
   @IBOutlet weak var mainView: NSView!
   @IBOutlet weak var betaIndicatorView: BetaIndicatorView!
+  @IBOutlet weak var lastFileContainerView: InitialWindowViewActionButton!
+  @IBOutlet weak var lastFileIcon: NSImageView!
+  @IBOutlet weak var lastFileNameLabel: NSTextField!
+  @IBOutlet weak var lastPositionLabel: NSTextField!
+  @IBOutlet weak var recentFilesTableTopConstraint: NSLayoutConstraint!
 
-  lazy var recentDocuments: [URL] = NSDocumentController.shared.recentDocumentURLs
+  lazy var recentDocuments: [URL] = {
+    NSDocumentController.shared.recentDocumentURLs.filter { $0 != lastPlaybackURL }
```

Comment:
I think this is weird. Users should be able to see the last file they played in history. They may want to watch that file from very beginning by clicking that file in history.

Reply:
It'll resume watch progress regardless of how you open the video. If I don't filter that there will be two completely duplicate records.

Reply:
If "Resume last playback position" is unchecked in Preferences, the files shouldn't resume progress. This change has made it impossible to play the last played file from the beginning.